core/src/main/scala/com/mesosphere/usi/models/package.scala

```diff
@@ -0,0 +1,45 @@
+package com.mesosphere.usi
+
+import com.mesosphere.usi.core.models._
+
+package models {
```

Comment:
Any reason this is not part of `core-models`?

Reply:
Can we move it after?

Reply:
I moved it. Don't forget to pull